tests/data/test_nypl_libraries.yml

```diff
@@ -0,0 +1,34 @@
+dataset:
+  name: &name test_nypl_libraries
+  version: "20210122"
+  acl: public-read
+  source:
+    url:
+      path: https://raw.githubusercontent.com/NYCPlanning/recipes/master/recipes/facdb/nypl_libraries.csv
```

Comment:
Why are we using this link to get the `test_nypl_libraries` data?

Reply:
oh yea that was the link in this template even before I renamed it. but it'd be nice to have that data in this repo so I'll add the csv here and changed the link in the two `test_nypl_libraries.yml` files